crates/nu-command/tests/commands/mut_.rs

```diff
@@ -177,3 +177,17 @@ fn mut_records_update_properly() {
     let actual = nu!(pipeline("mut a = {}; $a.b.c = 100; $a.b.c"));
     assert_eq!(actual.out, "100");
 }
+
+#[test]
+fn mut_value_with_if() {
+    let actual = nu!(pipeline("mut a = 3; $a = if 3 == 3 { 10 }; echo $a"));
```

Comment:
as the commands for these 4 tests are on single lines, i'm not sure we need to wrap them inside `pipeline` :thinking:

Reply:
Thanks for catching this!